Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -213,6 +314,8 @@ void IMixedRealityTouchHandler.OnTouchCompleted(HandTrackingInputEventData event
             {
                 touchPoints.Remove(eventData.Controller);
 
+                IsTouching = (touchPoints.Count > 0);
```

Comment:
Why directly set the properties here and not use the old method to determine IsTouching? touchPoints.Count > 0.

Reply:
hmm i'm not sure i see the previous change you're referring to. we used to set IsTouching in update which would be fine, but by setting it in the touch completed we can safe the per frame call of the setter. Am I missing something here?

Reply:
never mind.